model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
     templist = empty list of strings
```

Comment:
```suggestion
    templist = ""
```

To keep the original implementation, this should be an empty string. Not a list.